python/fledge/services/core/api/plugins/remove.py

```diff
@@ -152,7 +157,7 @@ def purge_plugin(plugin_type: str, name: str):
                             os.rmdir(os.path.join(root, name))
                 code = os.system('rm -rf {}'.format(path))
             else:
-                path = C_PLUGINS_PATH + '{}/{}'.format(plugin_type, name)
+                path = C_PLUGINS_PATH + '{}/{}'.format(plugin_type, org_name)
```

Comment:
Line 166:

code = os.system('rm -rf {}'.format(path))

As commented above code is 0 even for a wrong path
We do need a different check

Reply:
Resolved